Library/Formula/svtplay-dl.rb

```diff
@@ -22,6 +15,7 @@ class SvtplayDl < Formula
   end
 
   depends_on "rtmpdump"
+  depends_on "python"
```

Comment:
python in some older version of OS X  dont have support for SNI. those users using that old version get an error message while accessing the sites even that python in brew have support for it. 
after some digging we found out that brew change my shebang from "#!/usr/bin/env python" to "#!/usr/bin/python". 

How can i prevent that to happen?